examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<p>may be rephrase:  <code>We load the saved Merlin Models TensorFlow model.</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
+1